app/README.md

```diff
@@ -1,3 +1,5 @@
+What an amazing pull request.
```

Comment:
Let's put more emphasis on this description. If it's amazing then let's give it an exclamation mark!

Reply:
#is not